data/json/recipes/other/tools.json

```
@@ -878,7 +879,7 @@
     "difficulty": 3,
     "time": "20 m",
     "autolearn": true,
-    "book_learn": [ [ "fun_survival", 2 ], [ "manual_fabrication", 2 ] ],
+    "book_learn": [ [ "fun_survival", 2 ], [ "manual_fabrication", 2 ], [ "reference_fabrication1", 3 ] ],
```

```suggestion
    "book_learn": [ [ "fun_survival", 2 ], [ "manual_fabrication", 2 ], [ "reference_fabrication1", 2 ] ],
```

Autolearn already is at lvl 3